### PR TITLE
Fight_Game

### LINKED ISSUE #52: [New game]: Fight game
### 🎮 Game Request

A fighting game is a genre of video game in which a gamer battles against another character controlled by another gamer.
This fighting game feature special moves that are triggered through rapid sequences of button presses.

### Point down the features

Player 1: Controls:W A D Space F
Player 2: Controls:ArrowUp ArrowLeft ArrowRight ArrowDown

### Select program in which you are contributing

GSSoC23

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

### PR DESCRIPTION
**Pull request by S-ishita**
## PR Description 📜

Please include summary related to the issue you have fixed and describe your PR in brief over here by specifying the issue number on which you were working below
Fixes #52 

<hr>
 
## Mark the task you have completed ✅

- [x] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x] I have added screenshot for website preview in assets/images 
- [x] I have added entries for my game in main README.md
- [x] I have added README.md in my folder 
- [x] I have added working video of the game in README.md (optional)
- [x] I have specified the respective issue number for which I have requested the new game.

<hr>

## Add your screenshots(Optional) 📸

f
f1

--- 
<br>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

**Comment by rajatnai49** [comment]
Hey, @S-ishita please complete all the task which checklist you have given please read all the task and try to complete all of them, and when you complete all the given task then you can see your game on home page of the website like,

Screenshot from 2023-05-21 09-55-18

You can see given games, so you get idea that which are files you have to update!!

**Comment by S-ishita** [comment]
Hey @rajatnai49 . I updated the file now.Sorry for the mistake from my end.

**Comment by S-ishita** [comment]
- [x] I have added screenshot for website preview in assets/images
- [x] I have added entries for my game in main README.md
- [x] I have added README.md in my folder

File: Games/Fight_Game/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Fight-game
+
+#### Player 1 Controls:W A D Space F
+#### Player 2 Controls:ArrowUp ArrowLeft ArrowRight ArrowDown Control
```
Comment (kunjgit) [inline Games/Fight_Game/README.md:4]: add proper readme

File: README.md
Comment (kunjgit) [inline README.md:1]: just remove the number and add * in place of the number !!

**Review by kunjgit — CHANGES_REQUESTED** [review]

File: README.md
Comment (S-ishita) [reply README.md:1]: done

**Comment by kunjgit** [comment]
don't alter main index file and also make sure if you are adding game in main Readme you have to have your game under all existing games and you can have game number  as * we will add that

**Comment by S-ishita** [comment]
@kunjgit following changes have been made:

- undo the index.html file changes
- changes the number to * in readme

**Review by kunjgit — CHANGES_REQUESTED** [review]
make sure you don't alter main index file we have changed few things make it up to date to latest and don't alter it

**Review by kunjgit — APPROVED** [review]

**Comment by kunjgit** [comment]
Thank you  @S-ishita , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request merged**